<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedWindow</class>
 <widget class="QDialog" name="MedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MED MODE</string>
  </property>
  <widget class="QPushButton" name="leftButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>← </string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="powerButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QListWidget" name="mainMenuOptions">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>256</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rightButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>→</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="batteryStatus">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hintButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Help ?</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="intensityLevel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="skinElectrode">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>106</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Skin On/Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="intensityLevelLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity Level</string>
   </property>
  </widget>
  <widget class="QLabel" name="medTimerLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>MED timer (only operates on skin)</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="medTimer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>118</width>
     <height>26</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
